baas/src/main/scala/com/ing/baker/baas/http/APIRoutes.scala

```diff
@@ -26,7 +25,7 @@ object APIRoutes extends Directives with BaasMarshalling {
 
             val sensoryEventStatus = confirm.getOrElse(defaultEventConfirm) match {
               case "received"  => baker.processEventAsync(requestId, event).confirmReceived()
-              case "completed" => baker.processEventAsync(requestId, event).confirmCompleted()
+              case "completed" => baker.processEventAsync(requestId, event).confirmCompleted().sensoryEventStatus
```

Comment:
I would prefer if we can archieve this functionality without breaking backward comparability of the completed function.